src/modules/shared/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,16 @@
+import clsx from 'clsx';
+import { FC } from 'react';
+
+import styles from './Sidebar.module.scss';
+import { SidebarProps } from './Sidebar.types';
+
+const Sidebar: FC<SidebarProps> = ({ className, children, title, heading }) => {
+	return (
+		<section className={clsx(className, styles['c-sidebar'])}>
+			{heading ?? <b className={styles['c-sidebar__title']}>{title}</b>}
```

Comment:
Ik zou ipv een `<b>` gewoon al de juiste heading tag gebruiken.

Reply:
Hoe wordt dit hier dan aangepakt? `Account` lijkt me een `h2` te zijn, maar `Mijn mappen` een `h3`. `h2` als default en voor de `h3` een custom heading?

![image](https://user-images.githubusercontent.com/85545258/151949366-7f57473e-524a-450f-9084-2150aaf4a307.png)

Reply:
Good point, yes om de hiërarchie te bewaren by default `h2` en voor extra sidebars een custom heading.
Enkel zien dat de font sizes overeenkomen met het design 👍 .

Reply:
Als er een custom heading gebruikt wordt moet daar wel zelf styling op voorzien worden. In theorie kan `c-sidebar__title` wel hergebruikt worden uit `Sidebar.module.scss`